requirements/test.txt

```diff
@@ -12,14 +12,14 @@ expecttest ==0.3.0  # for test_ddp.py
 hypothesis ~=6.133.0  # for test_ddp.py
 numpy
 einops  # for test_einops.py
-litgpt==0.4.11  # for the model definition in tests and benchmarks
+litgpt==0.5.0  # for the model definition in tests and benchmarks  # todo: need update to latest
 absl-py # thunder/benchmarks/test_benchmark_litgpt.py
 pandas # thunder/benchmarks/test_benchmark_litgpt.py
 xlsxwriter # thunder/benchmarks/test_benchmark_litgpt.py
 jsonargparse # thunder/benchmarks/benchmark_litgpt.py
-bitsandbytes ==0.44.1; sys_platform != 'darwin'
-bitsandbytes ==0.42.0; sys_platform == 'darwin'
-transformers ==4.52.4 # for test_networks.py
+bitsandbytes>=0.44,<0.45; sys_platform!='darwin'
+bitsandbytes>=0.42,<0.43; sys_platform=='darwin'
```

Comment:
just relax it as a range 🐿️

Reply:
weird it was downloading 
```
#37 7.810 Collecting bitsandbytes==0.42.0 (from -r requirements/test.txt (line 20))
#37 7.814   Downloading bitsandbytes-0.42.0-py3-none-any.whl.metadata (9.9 kB)
```
two days ago on ARM linux

Reply:
First, apologies for breaking this for you.
I think we want `"arm" not in platform_machine` and `"arm" in platform_machine` as the conditions (cf. platform.machine() ). WDYT?

Reply:
Are you going to publish it to nvidia pypi?

Reply:
I'm not sure because those are dev cuda versions before public release.